test/color-test/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "color-test"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[dependencies]
+log = "0.4.0"
+log4rs = { path = "../.." }
+serde_yaml = { version = "0.9", optional = true }
```

Comment:
this shouldnt need a separate Cargo.toml, check out the Cargo.toml manifest docs around integration tests.